website/src/widgets/quickstart-install.js

```diff
@@ -9,23 +9,15 @@ const DEFAULT_PLATFORM = 'x86'
 const DEFAULT_MODELS = ['en']
 const DEFAULT_OPT = 'efficiency'
 const DEFAULT_HARDWARE = 'cpu'
-const DEFAULT_CUDA = 'cuda113'
+const DEFAULT_CUDA = 'cuda-wheel'
 const CUDA = {
     '8.0': 'cuda80',
     '9.0': 'cuda90',
     '9.1': 'cuda91',
     '9.2': 'cuda92',
     '10.0': 'cuda100',
     '10.1': 'cuda101',
-    '10.2': 'cuda102',
-    '11.0': 'cuda110',
-    '11.1': 'cuda111',
-    '11.2': 'cuda112',
-    '11.3': 'cuda113',
-    '11.4': 'cuda114',
-    '11.5': 'cuda115',
-    '11.6': 'cuda116',
-    '11.7': 'cuda117',
+    '10.2, CUDA 11.0+': 'cuda-wheel',
```

Comment:
```suggestion
    '10.2-11.x"': 'cuda-wheel',
```
So that `CUDA` doesn't get repeated in the displayed string?

Reply:
Repeating it was intentional since the range 10.2-11.x is so weird and unexpected in the context of the rest of this list. I think a user is going to be looking for "CUDA 11.".

Reply:
Can't we then just have an extra entry for CUDA 11.0+ that also maps to `cuda-wheel`?